posix.mak

```diff
@@ -431,10 +431,9 @@ ${GENERATED}/modlist-prerelease.ddoc : tools/modlist.d ${STABLE_DMD} $(DRUNTIME_
 		$(addprefix --dump , object std etc core dmd rt core.internal.array core.internal.util) >$@
```

Comment:
This looks like the problem @CyberShadow (`modlist-${LATEST}.ddoc` recipe has a different dmd/src path).

Reply:
Yes, well spotted!

Reply:
Actually, everything in the make files that uses `DMD_LATEST_DIR` needs to be aligned up with the use of `DMD_DIR`.

Reply:
Good idea, updated all the ones I could find.

Reply:
Looks much better now! :tada: